lib/themes/reps/css/reps.css

```diff
@@ -1909,8 +1904,8 @@ body.reps #mainContainer #layout-body #block-views-reps-videos-block .views-row
     padding-left: 10px;
   }
   /*REPR-1512 text should be forced to be under the main image. */
-  body.reps #mainContainer #layout-body #block-system-main .panel-body node-page .content .field-type-text-with-summary img.right,
-  body.reps #mainContainer #layout-body #block-system-main .panel-body .node-full .content div:not(.leaflet-map) img:first-child {
+  body.reps #mainContainer #layout-body #block-system-main .panel-body .node-page .content .field-type-text-with-summary img.right,
+  body.reps #mainContainer #layout-body #block-system-main .panel-body .node-full .content div:not(.leaflet-map, .file) img:first-child {
```

Comment:
FILE: .../lib/themes/reps/css/reps.css
----------------------------------------------------------------------
 1908 | ERROR | [x] Multiple selectors should each be on a single line

Reply:
@iulianweb ,
This is one selector div:not(.leaflet-map, .file) but the coma to separate classes to exclude is generating the error.  If it's really required I can adapt my less file and recreate the CSS.

Fyi: https://developer.mozilla.org/en-US/docs/Web/CSS/:not

Reply:
It's OK